README.md

```diff
@@ -152,3 +152,11 @@ example, another Thing was created in the meantime):
 $ curl http://localhost:8001/things
 {"things":[{"_id":"615f3313b231dd5cccf9c996","name":"answer","value":42},{"_id":"615f33abb231dd5cccf9c997","name":"spanner","value":18}]}
 ```
+
+### Generate SECRET KEY and add it in environment file
+Generate a new key for Authentication using following command:
+```
+$ openssl rand -hex 32
+```
+Store the generated key to .env file and give its path to docker-compose.yaml file.
+We have loaded secret key using FastAPI settings feature in api.auth
```

Comment:
Please move this to the `## Authentication` section as the secret key is required before being able to create user tokens.